### DISCUSSION TITLE
Follow page rating

### DISCUSSION THREAD

**Discussion by mquandalle** [plausible/analytics#770] [Anything Else]
I have a page rating system on my site that is really useful to collect user feedback. I would like to see the average rating of each page in Plausible (and ideally the distribution of rates as well). 

![image](https://user-images.githubusercontent.com/1730702/109520054-1a59fa00-7aac-11eb-91ec-39c7b02214c2.png)

(Each answer is associated with a rate 0 / 0.3 / 0.6 / 1)

As far as I know it is not possible currently to aggregate custom events attributes in Plausible

**Comment by metmarkosaric** [comment]
thanks for the feedback @mquandalle! true, it's not possible to aggregate custom event attributes at this moment. there has been a request to add monetary values to events (#256) so perhaps something like that would be able to work for your use case too
